tests/testing_harness.py

```diff
@@ -66,9 +66,14 @@ def update_results(self):
             self._cleanup()
 
     def _run_openmc(self):
-        if config['mpi']:
+        if config['mpi'] and not config['event']:
             mpi_args = [config['mpiexec'], '-n', config['mpi_np']]
             openmc.run(openmc_exec=config['exe'], mpi_args=mpi_args)
+        elif config['mpi'] and config['event']:
+            mpi_args = [config['mpiexec'], '-n', config['mpi_np']]
+            openmc.run(openmc_exec=config['exe'], mpi_args=mpi_args, event_based=True)
+        elif config['event']:
+            openmc.run(openmc_exec=config['exe'], event_based=True)
```

Comment:
Seems like the different cases are equivalent to passing `event_based=config['event']`, as far as that argument is concerned

Also, can you add a default `False` value in `tests/regression_tests/__init__.py`?